### DISCUSSION TITLE
Make Mutagenic Serum a valid replacement for regular Mutagen

### DISCUSSION THREAD

**Discussion by RodriguesCIA** [cataclysmbn/Cataclysm-BN#2070] [General]
If the player plans on being conservative with their mutagen use, the highly random nature of Mutagenic Serum may not sound too appealing, and even for players who drink it like gatorade, it becomes increasingly less useful as targeted high level mutagen becomes available.

And yet still, you are likely to find lots of Mutagenic Serum in labs, that, in these cases, have no much use.

How about making it a valid replacement for 2 doses of regular Mutagen in advanced mutagen recipes? Or a recipe to dilute it into 2 regular mutagens.

**Comment by Coolthulhu** [comment]
Could work.
Still, I'd rather give it a specific benefit, like making it more likely to get good mutations compared to other types of mutagens and serums.
